students/byeonghwi/.gitignore

```
@@ -0,0 +1,398 @@
+
```

- 병휘님 현재 Repository에는 제가 이미 추가해놓은 .`gitignore`에 의해서 환경 변수를 관리하는 파일인 `my_settings.py`가 git에 추적되지 않아 올라오지 않지만 push하신 내용대로면 Repository에 추가되었을 겁니다.
- 따라서 항상 환경변수 파일이 `.gitignore`에 담겨있을 수 있도록 설정해주세요.
```suggestion
my_settings.py
```

---

`.gitignore` 에 `my_settings.py ` 추가 완료했습니다